nitk-thesis.csl

```diff
@@ -0,0 +1,234 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+  <info>
+    <title>NITK Thesis</title>
```

Comment:
We usually don't allow for acronyms in titles. Please change this to `<title>National Institute of Technology Karnataka</title>`